compiler-rt/lib/rtsan/rtsan_flags.cpp

```diff
@@ -35,6 +35,7 @@ void __rtsan::InitializeFlags() {
   {
     CommonFlags cf;
     cf.CopyFrom(*common_flags());
+    cf.exitcode = 43; // (TR-)808 % 255 = 43
```

Comment:
please remove the comment, that just confuses people. add something like `// arbitrarily chosen`?

Reply:
Done, thanks for the feedback, agreed :)